src/views/v2/day/time-separator.php

```diff
@@ -23,17 +23,17 @@
 	return;
 }
 
-$event_start_datetime = strtotime( tribe_get_start_date( $event->ID, true, Tribe__Date_Utils::DBDATETIMEFORMAT ) );
-$event_start_hour = strtotime( date( 'Y-m-d H:00:00', $event_start_datetime ) );
+$event_start_datetime = strtotime( $event->dates->start->format( Date::DBDATETIMEFORMAT ) );
```

Comment:
For consistency with what we do in other code fragments, and to avoid confusion, change the alias to `Dates` (plural).

Adding here as I cannot do that on line 17.